### PR TITLE
refactor(test): refactor manifest setup in stack tests 

### PR DESCRIPTION
**Pull request by Lou1415926**
This PR refactors the manifest setup in stack for backend svc tests.

In addition, it adds entrypoint/command fields for backend svc and scheduled job (as a follow up on #1999).

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: internal/pkg/deploy/cloudformation/stack/backend_svc_test.go [hunk 1]
```diff
@@ -79,17 +57,24 @@ func TestBackendService_Template(t *testing.T) {
...
 				m.EXPECT().Read(desiredCountGeneratorPath).Return(&template.Content{Buffer: bytes.NewBufferString("something")}, nil)
 				svc.parser = m
 				svc.addons = mockTemplater{err: errors.New("some error")}
 			},
-			wantedErr: fmt.Errorf("generate addons template for %s: %w", aws.StringValue(testBackendSvcManifest.Name), errors.New("some error")),
+			wantedErr: fmt.Errorf("generate addons template for %s: %w", testName, errors.New("some error")),
 		},
 		"failed parsing sidecars template": {
-			manifest: testBackendSvcManifestWithBadSidecar,
+			setUpManifest: func(svc *BackendService) {
```
Comment (efekarakus) [inline internal/pkg/deploy/cloudformation/stack/backend_svc_test.go:69]: nice, i really like this change!

File: internal/pkg/deploy/cloudformation/stack/backend_svc_test.go [hunk 2]
```diff
@@ -222,6 +243,21 @@ Outputs:
 }
 
 func TestBackendService_Parameters(t *testing.T) {
+	var testBackendSvcManifest = manifest.NewBackendService(manifest.BackendServiceProps{
```
Comment (efekarakus) [inline internal/pkg/deploy/cloudformation/stack/backend_svc_test.go]: nit:
```suggestion
	testBackendSvcManifest := manifest.NewBackendService(manifest.BackendServiceProps{
```

File: internal/pkg/deploy/cloudformation/stack/backend_svc_test.go [hunk 3]
```diff
@@ -27,45 +27,23 @@ var (
...
-	Port: 8080,
-	HealthCheck: &manifest.ContainerHealthCheck{
-		Command:     []string{"CMD-SHELL", "curl -f http://localhost/ || exit 1"},
-		Interval:    &testInterval,
-		Retries:     &testRetries,
-		Timeout:     &testTimeout,
-		StartPeriod: &testStartPeriod,
-	},
-})
+	testName       = "frontend"
```
Comment (efekarakus) [inline internal/pkg/deploy/cloudformation/stack/backend_svc_test.go]: nit:
```suggestion
	testServiceName       = "frontend"
```

**Review by efekarakus — APPROVED** [review]
✨

**Review by iamhopaul123 — APPROVED** [review]
Nice! Feel free to take off the label once address those comments.

**Pull request merged**